src/ASM/ASMs2D.C

```diff
@@ -1297,44 +1297,40 @@ bool ASMs2D::updateCoords (const Vector& displ)
 }
 
 
-void ASMs2D::getBoundaryNodes (int lIndex, IntVec& nodes) const
+void ASMs2D::getBoundaryNodes (int lIndex, IntVec& nodes, int basis) const
 {
-  if (!surf) return; // silently ignore empty patches
+  if (basis == 0)
+    basis = 1;
+
+  if (!getBasis(basis)) return; // silently ignore empty patches
```

Comment:
vennligst bruk this-> så er det tydeligere at getBasis er en member og ikke en global static funksjon (konvensjon jeg har lagt meg på ellers). Dessuten står det over at basis==0 betyr for alle basiser. Hvordan forklares det da at den her resettes til 1? Da vil vel man aldri kunne få ut for alle basisene (om nå det skulle være ønskelig).
Eller er doxyen feil?